### PR TITLE
e2e: fixes test External service add, edit, delete

### PR DESCRIPTION
**Pull request by uwedeportivo**
fixes https://buildkite.com/sourcegraph/e2e/builds/10000#3af1e45e-7de1-4ea9-b5c1-3dce9bd2f906

### CODE REVIEW

**Comment by uwedeportivo** [comment]
never mind, fix didn't work: https://buildkite.com/sourcegraph/e2e/builds/10001#7debbb41-b7d7-4919-abb6-0557d983ef97

**Comment by uwedeportivo** [comment]
https://buildkite.com/sourcegraph/e2e/builds/10003#0cb2c97e-188a-4b8f-9873-baae6a6eda72

this worked

File: web/src/end-to-end/end-to-end.test.ts [hunk 1]
```diff
@@ -213,10 +213,10 @@ describe('e2e test suite', () => {
 
             // Type in a new external service configuration.
             await driver.replaceText({
-                selector: '.monaco-editor',
+                selector: '.e2e-external-service-editor .monaco-editor',
                 newText:
                     '{"url": "https://github.myenterprise.com", "token": "second-token", "repositoryQuery": ["none"]}',
-                selectMethod: 'keyboard',
+                selectMethod: 'selectall',
```
Comment (felixfbecker) [inline web/src/end-to-end/end-to-end.test.ts:219]: The comment here claims that `selectall` doesn't work in the Monaco editor:

https://github.com/sourcegraph/sourcegraph/blob/ccce422e6192807088ca686566a8ee178d1c14ba/shared/src/testing/driver.ts#L49

Is this false? Or is it maybe platform-dependent?

Comment (uwedeportivo) [reply web/src/end-to-end/end-to-end.test.ts:219]: no clue. i know that these code changes make it green

**Pull request merged**